ItemDTO/pom.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.7.0</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>jar</packaging>
+    <groupId>com.visma</groupId>
+    <version>1.0-SNAPSHOT</version>
+    <artifactId>ItemDTO</artifactId>
```

Comment:
gal nelabai prie vienos klases reiketu ristis, nes o jei sugalvosim dar koki api kur grazins kitoki objekta?
be to reiketu visus modulius pervadinti i lowercase.
gal koks  'warehouse-dto` geriau